acceptance/tests/cloud/basic_test.go

```diff
@@ -229,6 +229,57 @@ func TestBasicCloud(t *testing.T) {
 
 	logger.Log(t, "creating static-server deployment")
 	k8s.DeployKustomize(t, ctx.KubectlOptions(t), cfg.NoCleanupOnFailure, cfg.DebugDirectory, "../fixtures/bases/static-server")
-	// time.Sleep(1 * time.Hour)
-	// TODO: add in test assertions here
+	t.Log("Finished deployment. Validating expected conditions now")
+	// Give some time for collector send metrics
+	time.Sleep(5 * time.Second)
```

Comment:
I could see this causing flakes, especially in CI as sometimes these machines are slooow.  Would a retry be better?